.travis.yml

```diff
@@ -155,7 +155,7 @@ jobs:
     - stage: deploy
       before_install: mkdir -p $HOME/.bin; export PATH=$PATH:$HOME/.bin; .travis/before_install.sh
       install: .travis/install.sh
-      script: RAIDENVERSION=$TRAVIS_TAG make bundle-docker
+      script: .travis/set_tag.sh; make bundle-docker
```

Comment:
That won't work. The variable will be set in a subshell and vanish once the script is finished executing. You need to source (either with the `source` or the `.` operator) the script for it to be able to set variables in the current shell context.

Also please use another variable besides $TRAVIS_TAG in case of a nightly build. The reason is that `on: tags: true` check below also checks against that variable.

Reply:
I have assigned the RAIDENVERSION variable in the ``set_tag.sh`` (using: `` . .travis/set_tag.sh`` ) and I will change 
- This ``- dist/raiden-${TRAVIS_TAG}-linux smoketest`` to ``- dist/raiden-${RAIDENVERSION}-linux smoketest`` 
and 
- This `` cd dist && zip -9 -r raiden-${TRAVIS_TAG}-macOS.zip raiden* `` to ``cd dist && zip -9 -r raiden-${RAIDENVERSION}-macOS.zip raiden*`` 
since is not gonna change that much the behavior, right? 

Also the script is gonna be edited like that: 
```
if $TRAVIS_TAG; then
    $RAIDENVERSION=$result
else
    DATE=`date +%Y.%m.%d`
    $RAIDENVERSION="nightly-$DATE"
fi
```